### PR TITLE
fix wrong vad notification to background bargin

### PR DESCRIPTION
**Pull request by xquanluu**
#767 

The gather fire both 2 events **vad** and **resolve** by transcription to background bargin that make bargin clean up verbs in queue twice

### CODE REVIEW

**Comment by xquanluu** [comment]
Hi @davehorton 
They confirmed that the PR fixed their issue.

**Review by davehorton — COMMENTED** [review]
are there any scenarios where we need to reset this.bargeInHandled to false?  Like in order to reset for a second barge in?

File: lib/utils/background-task-manager.js [hunk 1]
```diff
@@ -186,6 +186,8 @@ class BackgroundTaskManager extends Emitter {
   }
 
   _bargeInTaskCompleted(evt) {
+    if (this.bargeInHandled) return;
+    this.bargeInHandled = true;
```
[the remark this answers is not in the document]
Comment (xquanluu) [reply lib/utils/background-task-manager.js]: fixed

**Review by davehorton — APPROVED** [review]

**Comment by paulotelles** [comment]
the issue was fixed, here the call sids tested:

call_sid:
4627bb06-4b74-4cbb-8294-e1f05a8dce71

call_sid:
fc821df4-9989-479e-83d9-e29f2160b52c

**Pull request merged**